### PR TITLE
Feature/7 landing page blueprint

### PR DESCRIPTION
**Pull request by ashaduzzamane**
### What does this implement/fix? Explain your changes.
This creates a blueprint for the landing site. Incorporating GatsbyJs

### Does this fix any currently open issues? 
n/a

### Any relevant logs, error output, etc?
n/a

### Any other comments? Questions for reviewers?
n/a

### Where has this been tested?
**Platforms (Desktop, Phone, Tablet, etc...)**
Desktop

**Browsers**

### Checklist
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/components/footer.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+import React from "react"
+
+import * as Styles from "../styles/components/index.js"
+
+
+const Footer = () => (
+    <div class="nav"></div>
```
Comment (alinastarkov) [inline src/components/footer.js]: className instead of class

File: src/pages/404.js [hunk 2]
```diff
@@ -1,14 +1,9 @@
...
-import SEO from "../components/seo"
-
 const NotFoundPage = () => (
-  <Layout>
-    <SEO title="404: Not found" />
-    <h1>NOT FOUND</h1>
-    <p>You just hit a route that doesn&#39;t exist... the sadness.</p>
-  </Layout>
+  <div class="notfound">
+    <a> NOT FOUND!</a>
```
Comment (alinastarkov) [inline src/pages/404.js]: link with anchor? If no link then use text element instead

File: src/components/sponsor.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    <h1>A tagline about sponsorship</h1>
+
+    <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse purus metus, tincidunt at finibus quis, 
+        <br />
+        <br />
+        Quisque nisl eros, convallis ut pulvinar euismod, accumsan sit amet tortor. Aliquam ligula mi, interdum in 
+        tempor at, semper in tellus. Aenean pellentesque urna tempus dolor scelerisque, id porttitor mi dictum. Vestibulum convallis odio et magna aliquam, sed sollicitudin dui tempus. 
+    </p>
+    </Styles.Sponsor>
```
[the remark this answers is not in the document]
Comment (ashaduzzamane) [reply src/components/sponsor.js]: yea. was just testing the functionality

File: src/pages/index.js [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
-    <p>Welcome to your new Gatsby site.</p>
-    <p>Now go build something great.</p>
-    <div style={{ maxWidth: `300px`, marginBottom: `1.45rem` }}>
-      <Image />
+    <div>
+        <Nav />
+        <Hero />
+        <About />
+        <Sponsor />
+        <Footer />
```
[the remark this answers is not in the document]
Comment (ashaduzzamane) [reply src/pages/index.js]: i thought this was the order according to what we were given in figma

**Comment by cvrrie** [comment]
> 1. Feedbacks on small things like className, .
> 2. Delete package-lock.json, be careful with this file, you shouldnt commit it on the branch
> 3. Fixed conflicts, will commit the changes after my branch is merged.

Maybe add package-lock.json to the .gitignore file?

File: src/pages/index.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (loreina) [reply src/pages/index.js]: I think import order should match page order

File: src/components/nav.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+
+import React from "react"
+
+import * as Styles from "../styles/components/index.js"
+
+
+const Nav = () => (
+        <div class="nav">
```
[the remark this answers is not in the document]
Comment (loreina) [reply src/components/nav.js]: I think we can push this to when @cvrrie works on navigation

File: src/pages/404.js (hunk 2 above)
Comment (loreina) [reply src/pages/404.js]: Let's also push this to when we work on 404 page

**Pull request merged**